pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ">=0.15.7 <0.18.0"
+  intl: "^0.18.0"
```

Comment:
Lets do `>=0.15.7 <1.0.0` and hope they follow proper semantic versioning

Reply:
I think we can remove this in the next release if we just expect a locale tag instead of a locale object

Reply:
According to their versioning doc:

Although semantic versioning doesn’t promise any compatibility between versions prior to 1.0.0, the Dart community convention is to treat those versions semantically as well. The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public API. For simplicity’s sake, avoid using + after the version reaches 1.0.0.

https://dart.dev/tools/pub/versioning#semantic-versions

so with that reasoning I think >=0.15.7 <0.19.0 would be the safest bet since 0.20 could be considered a breaking change deployment